docker/dirigible/templates/Dockerfile.template

```diff
@@ -0,0 +1,9 @@
+FROM dirigiblelabs/dirigible-all AS build
```

Comment:
Can we use a version tag on the builder image here? The idea is hermetically close this version when releasing this container.

Reply:
Yes! I changed to latest tag.